flaskbb/templates/message/conversation_list.html

```diff
@@ -41,9 +41,20 @@
                 </div>
                 <!-- meta info (date, user) -->
                 <div class="conversation-meta">
-                    From <a href="{{ conversation.from_user.url }}">{{ conversation.from_user.username }}</a>
-                    to <a href="{{ conversation.to_user.url }}">{{ conversation.to_user.username }}</a>
-                    on {{ conversation.last_message.date_created|format_date("%d %B %Y - %H:%M") }}
+                    {% trans %}From{% endtrans %}
```

Comment:
Maybe this is me, but it seems like we should use interpolation with gettext would be better. Not a big deal either way to me though.

Reply:
Yep, you are right 👍